FantasmoSDK/app/src/test/java/com/fantasmo/sdk/config/RemoteConfigTest.kt

```diff
@@ -43,4 +50,16 @@ class RemoteConfigTest {
             imageQualityFilterModelVersion = "0.1.0"
         )
     }
+    @Test
+    fun testRemoteConfigNullId() {
```

Comment:
and here the same, something like `fun testUseStoredConfigWhenRemoteConfigIdIsNull`